### DISCUSSION TITLE
Separate editing mode on by default for new users?

### DISCUSSION THREAD

**Discussion by samstride** [outline/outline#7158] [General] [answered]
Hi,

When a new user is created, we would like to default certain user settings, for example: `Separate editing` is on by default.

Also, is there any way for admins to turn on `Separate editing` for existing users?

The reason I ask is that we have users opening a documenting and accidentally editing it.

Thanks.

**Comment by tommoor** [comment]
Yes, the team setting will be used as the default for new users

**Comment by samstride** [comment]
@tommoor , how do I turn on `separate editing` for a team?

We are on v0.77.2, self hosted.

**Reply by tommoor** [reply, marked as answer]
Settings -> Features -> Separate editing

**Reply by samstride** [reply]
Sweet, thanks for the quick response.
